programs/mmm/src/errors.rs

```diff
@@ -72,4 +72,12 @@ pub enum MMMErrorCode {
     InvalidTokenExtension, // 0x1791
     #[msg("Unsupported asset plugin")]
     UnsupportedAssetPlugin, // 0x1792
+    #[msg("Mismatched creator data lengths")]
+    MismatchedCreatorDataLengths, // 0x1793
+    #[msg("Invalid cnft creators")]
+    InvalidCnftCreators, // 0x1794
```

Comment:
this is not used, can probably remove - also, can we merge some of these errors with the existing ones?